app/src/main/java/org/p2p/wallet/newsend/interactor/SendViaLinkInteractor.kt

```diff
@@ -0,0 +1,18 @@
+package org.p2p.wallet.newsend.interactor
+
+import java.math.BigInteger
+import org.p2p.core.token.Token
+import org.p2p.solanaj.core.PublicKey
+
+class SendViaLinkInteractor(
+    private val sendInteractor: SendInteractor
+) {
+
+    fun createSendTransaction(
+        destinationAddress: PublicKey,
+        token: Token.Active,
+        lamports: BigInteger
```

Comment:
we have lamports in Token.Active or is it different lamports?

Reply:
Ofc it's different